stories/control/legend.stories.tsx

```diff
@@ -35,10 +36,25 @@ const data2 = [
 ];
 
 export function Control(props: LegendProps) {
+  const [highlight, setHighlight] = useState<boolean>(false);
   return (
     <Plot {...DEFAULT_PLOT_CONFIG}>
-      <Legend {...props} />
-      <LineSeries data={data1} xAxis="x" yAxis="y" label="Label line series" />
+      <Legend
+        {...props}
+        onClick={() => {
+          setHighlight((highlight) => !highlight);
+        }}
+        style={{ fontWeight: highlight ? 'bold' : 'normal' }}
```

Comment:
Please rename this to `labelStyle` and pass the style to each label, not to the `<g>` element

Reply:
done